### PR TITLE
Clarify that declarative event mapping converts event names to lowercase.

### PR DESCRIPTION
**Pull request by arthurevans**
Includes the changes from PR #873 and a few other fixes.

### CODE REVIEW

**Comment by arthurevans** [comment]
@innubili I've added your changes in here.
@ebidel Minor updates to your article, are you OK with them?

Thanks!

File: articles/communication.md [hunk 1]
```diff
@@ -417,22 +417,28 @@ on `<say-bye>`.
...
 Parts of your app or other {{site.project_title}} elements can declare a `<core-signals>`
 element to catch the named signal:
 
 {%raw%}
-    <core-signals on-core-signal-foo="{{fooSignal}}"></core-signals>
+    <core-signals on-core-signal-new-data="{{newData}}"></core-signals>
 {%endraw%}
 
+**Lowercase event names.** When you use a declarative handler, the event name 
+is convered to lowercase, because attributes are case-insensitive.
```
Comment (ebidel) [inline articles/communication.md]: converted

**Comment by ebidel** [comment]
LGTM. Just the typo.

**Comment by addyosmani** [comment]
@ebidel fixed the typo in branch. Merging away now. Thanks @arthurevans!

**Pull request merged**